composer.json

```diff
@@ -17,7 +17,8 @@
 	],
 	"require": {
 		"php": ">=5.3.3",
-		"composer/installers": "~1.0"
+		"composer/installers": "~1.0",
+		"monolog/monolog": "~1.15"
```

Comment:
Are we sure this is the right version requirement? Are you relying on features that were added between 1.0 and 1.15?